include/SolutionOptions.h

```diff
@@ -232,6 +234,7 @@ class SolutionOptions
 
   bool newHO_;
 
+  bool resetTAMSAverages_;
```

Comment:
@sayerhs, not sure I follow. There was a naming inconsistency that I fixed (see comment below). This bool is needed for TAMS and SST with TAMS

Reply:
I think what he means is having this option in here instead of in the general `solution_options`

```        
        - TAMS:
            name: myTAMS
            max_iterations: 1
            convergence_tolerance: 1e-8

Reply:
This will have to wait further refinement until Jeremy and I figure out how to initialize mdot in TAMS instead of SST.